x-pack/platform/plugins/shared/entity_manager/server/lib/v2/definitions/get_identity_fields.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EntityV2 } from '@kbn/entities-schema';
+import { Logger } from '@kbn/core/server';
+import { readSourceDefinitions } from './source_definition';
+import { InternalClusterClient } from '../types';
+import { UnknownEntityType } from '../errors/unknown_entity_type';
+import { InvalidEntityInstance } from '../errors/invalid_entity_instance';
+
+export async function getIdentityFields(
```

Comment:
```suggestion
export async function identityFieldsBySource(
```

While you might want to name functions as verbs in general, for things that from the outside seem to only produce data (getters, data accessors) it's usually more helpful to name them as if they were properties of the class instead (nouns).

This is also reflected in how you name the variable you're returning here, that is the goal of this method, the "getting" part isn't really important (since it means you also have to define what you mean by "get", compared to other fancy words like compute, combine, blabla).

Imagine the usage code:
```
const identityFieldsBySource = entityManager.v2.identityFieldsBySource(type, ...);
```

Reply:
renamed